### PR TITLE
Ignore case for search

### LINKED ISSUE #676: Ignore case for search
**Idea Description**
If we want to search Compose. We can't search `compose`. We should search `compose`.
Can we make it ignore case?

**Reference images and links**
image

### PR DESCRIPTION
**Pull request by CookieySun**
## Issue
- close #676

## Overview (Required)
- Ignore case for search
- Ignore case for Highlight

## Links
- 

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt [hunk 1]
```diff
@@ -62,7 +62,7 @@ public data class Timetable(
         }
         if (filters.searchWord.isNotBlank()) {
             timetableItems = timetableItems.filter { timetableItem ->
-                timetableItem.title.currentLangTitle.contains(filters.searchWord)
+                timetableItem.title.currentLangTitle.lowercase().contains(filters.searchWord.lowercase())
```
Comment (takahirom) [inline core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt:65]: Can you check it?

```
Step 'ktlint' found problem in 'src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt':

Error on line: 65, column: 1
Exceeded max line length (100)
java.lang.AssertionError: Error on line: 65, column: 1
Exceeded max line length (100)
```

Comment (CookieySun) [reply core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt]: I fix it:pray:

File: core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt [hunk 2]
```diff
@@ -62,7 +62,9 @@ public data class Timetable(
         }
         if (filters.searchWord.isNotBlank()) {
             timetableItems = timetableItems.filter { timetableItem ->
-                timetableItem.title.currentLangTitle.contains(filters.searchWord)
+                timetableItem.title.currentLangTitle.lowercase().contains(
```
Comment (takahirom) [inline core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt:65]: How about using contains(text, ignoreCase = false)?

**Comment by takahirom** [comment]
Looks good! Please take a look at small point🙏

File: core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt (hunk 2 above)
Comment (CookieySun) [reply core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt]: I don't know this option.
But, false is case sensitive. so I use true.

**Review by takahirom — APPROVED** [review]
Thanks!

**Pull request merged**